### PR TITLE
sdcv, visudo: add link and --version example

### PR DESCRIPTION
**Pull request by werewolf-65**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/visudo.md [hunk 1]
```diff
@@ -9,3 +10,7 @@
 - Check sudoers file for errors:
```
Comment (CleanMachine1) [inline pages/common/visudo.md:9]:
```suggestion 
- Check the sudoers file for errors:
```

File: pages/common/visudo.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # visudo
 
 > Safely edit the sudoers file.
+> More information: <https://manned.org/visudo>.
 
 - Edit sudoers file:
```
Comment (CleanMachine1) [inline pages/common/visudo.md:5]:
```suggestion 
- Edit the sudoers file:
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

File: pages/common/visudo.md [hunk 3]
```diff
@@ -1,11 +1,16 @@
 # visudo
 
 > Safely edit the sudoers file.
+> More information: <https://manned.org/visudo>.
```
Comment (marchersimon) [inline pages/common/visudo.md:4]: We should use the official website here:
```suggestion
> More information: <https://www.sudo.ws/man/1.8.13/visudo.man.html>.
```

**Review by marchersimon — APPROVED** [review]
Thanks @werewolf-65

**Comment by marchersimon** [comment]
@mfrw @CleanMachine1 could you please review this?

**Review by sbrl — APPROVED** [review]
Thanks, @werewolf-65!

**Comment by sbrl** [comment]
Merging, as this has been sitting around a while and all comments appear to have been addressed.

**Comment by werewolf-65** [comment]
> Merging, as this has been sitting around a while and all comments appear to have been addressed.

Thanks!

**Pull request merged**